package.json

```diff
@@ -25,7 +25,7 @@
 
         "less": "~1.7.0",
         "csso": ">=1.3.11 <2.0.0",
-        "requirejs": "~2.1.14",
+        "requirejs": "~2.2.0",
```

Comment:
yeah for now at least. `make` uses the npm version as it's in npm's path, but all of our references in js and html point to bower_components. shouldn't be hard to make it only use the npm version though